start-postgres-elasticsearch-with-docker.sh

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env bash
```

Comment:
Self Review:

I adapted this from a script we use to start containers locally at work for local testing and CI.